engines/hdb/configure.engine

```diff
@@ -1,3 +1,3 @@
 # This file is included from the main "configure" script
-# add_engine [name] [desc] [build-by-default] [subengines] [base games] [deps]
-add_engine hdb "Hyperspace Delivery Boy!" yes "" "" "16bit highres lua"
+# add_engine [name] [desc] [build-by-default] [subengines] [base games] [deps] [components]
+add_engine hdb "Hyperspace Delivery Boy!" yes "" "" "16bit highres" "lua"
```

Comment:
Shouldn't Lua still be a dependency rather than an optional component?

Reply:
Oh, truly so. This existed before I made components and features interchangeable. Fixed.